DQMOffline/Trigger/plugins/MssmHbbBtagTriggerMonitor.cc

```diff
@@ -0,0 +1,175 @@
+#include <math.h>
+
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "Geometry/Records/interface/TrackerTopologyRcd.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DQM/TrackingMonitor/interface/GetLumi.h"
+#include "CommonTools/TriggerUtils/interface/GenericTriggerEventFlag.h"
+
+#include "DataFormats/Common/interface/Handle.h"
+
+
+#include "DQMOffline/Trigger/plugins/MssmHbbBtagTriggerMonitor.h"
```

Comment:
looking at the code, it seems to be enough general
I would suggest to update the name of the module

in addition, I think BTV might exploit your development as well !
@natalia-korneeva @JyothsnaKomaragiri @XavierAtCERN @heppye @defranchis